### PR TITLE
add core address and background color

### PR DESCRIPTION
**Pull request by chrouzz**
add a background color to the farm space and another core wallet.

### CODE REVIEW

**Review by e-asia — APPROVED** [review]

**Comment by chrouzz** [comment]
thank you for quick merge, look like the bg-color didn't work, it still white right now. Any idea why?
image

**Comment by chrouzz** [comment]
> thank you for quick merge, look like the bg-color didn't work, it still white right now. Any idea why?
> image

oups, it was only approved not merge yet my bad

**Comment by chrouzz** [comment]
How long does it usually take to merge the approved change?

**Comment by e-asia** [comment]
@chrouzz This will be done in the next few hours

**Comment by chrouzz** [comment]
> @chrouzz This will be done in the next few hours

great, thanks

File: spaces/farm/skin.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.farm{
```
Comment (bonustrack) [inline spaces/farm/skin.scss:1]:
```suggestion
.farm {
```

**Pull request merged**